tgui/packages/tgui/interfaces/TachyonArray.js

```diff
@@ -0,0 +1,108 @@
+import { useBackend } from '../backend';
+import { Button, Flex, Table, NoticeBox, Section, LabeledList } from '../components';
+import { Window } from '../layouts';
+
+export const TachyonArray = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    records = [],
+    explosion_target,
+    toxins_tech,
+    printing,
+  } = data;
+  return (
+    <Window>
+      <Window.Content scrollable>
+        <Section>
+          <LabeledList>
+            <LabeledList.Item label="Shift's Target">
+              {explosion_target}
+            </LabeledList.Item>
+            <LabeledList.Item label="Current Toxins Level">
+              {toxins_tech}
+            </LabeledList.Item>
+            <LabeledList.Item label="Administration">
+              <Button
+                icon="print"
+                content="Print All Logs"
+                disabled={!records.length || printing}
+                align="center"
+                onClick={() => act('print_logs')} />
+              <Button.Confirm
+                icon="trash"
+                content="Delete All Logs"
+                disabled={!records.length}
+                color="bad"
+                align="center"
+                onClick={() => act('delete_logs')} />
+            </LabeledList.Item>
+          </LabeledList>
+        </Section>
+        {!records.length ? (
+          <NoticeBox>
+            No Records
+          </NoticeBox>
+        ) : (<TachyonArrayContent />)}
```

Comment:
```suggestion
        ) : (
          <TachyonArrayContent />
        )}
```

Reply:
The formatter kinda shat itself when I did that. I'll change it to this though